bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -131,12 +131,8 @@ spec:
                                 description: The key of the secret to select from.  Must be a valid secret key.
                                 type: string
                               name:
-                                default: ""
```

Comment:
Is this change intentional ? (I don't see any change in api that would trigger this update 👀 )

Reply:
Unit tests are also failing because of it

Reply:
must be the sdk version

Reply:
not happening locally yet again :/